site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -238,6 +238,97 @@ Apply an Annotation with the prefix "agones.dev/sdk-" to the backing `GameServer
 curl -d '{"key": "foo", "value": "bar"}' -H "Content-Type: application/json" -X PUT http://localhost:${AGONES_SDK_HTTP_PORT}/metadata/annotation
 ```
 
+### Counters
+
+{{< alpha title="Counters" gate="Counters" >}}
+
+#### Alpha: GetCounter
+This function retrieves a specified counter by its name and returns its information.
+
+##### Example
+
+```bash
+curl -H "Content-Type: application/json" -X GET http://localhost:${AGONES_SDK_HTTP_PORT}/v1alpha1/counters/conformanceTestCounter
+```
+
+Response:
+```json
+{"name":"conformanceTestCounter", "count":"1", "capacity":"10"}
+```
+
+#### Alpha: UpdateCounter
+This function updates the specified counter's properties, such as its count and capacity, and returns the updated counter details.
+
+##### Example
+
+```bash
+curl -d '{"count": "5", "capacity": "11"}' -H "Content-Type: application/json" -X PATCH http://localhost:${AGONES_SDK_HTTP_PORT}/v1alpha1/counters/conformanceTestCounter
+```
+
+Response:
+```json
+{"name":"conformanceTestCounter", "count":"5", "capacity":"11"}
+```
+
+### Lists
+
+{{< alpha title="Lists" gate="Lists" >}}
+
+#### Alpha: GetList
+This function retrieves a specific list identified by its name, returns the list's information.
+
+##### Example
+```bash
+curl -H "Content-Type: application/json" -X GET http://localhost:${AGONES_SDK_HTTP_PORT}/v1alpha1/lists/conformanceTestList
+```
+
+Response:
+```json
+{"name":"conformanceTestList", "capacity":"100", "values":["test0", "test1", "test2"]     
+```
+
+#### Alpha: UpdateList
+This function updates the specified list's properties, such as its capacity and values, returns the updated list details.
```

Comment:
```suggestion
This function updates the specified list's properties, such as its capacity and values, returns the updated list details. This will overwrite all existing List.Values with the update list request values. Use addValue or removeValue for modifying the List.Values field.
```